packages/fast-usdc/src/utils/deploy-config.js

```diff
@@ -46,6 +46,169 @@ const agoricAssetInfo = defaultAssetInfo.filter(
 const DepositForBurnEvent =
   'DepositForBurn(uint64,address,uint256,address,bytes32,uint32,bytes32,bytes32)';
 
+/** @satisfies {Record<string, CosmosChainInfo & Passable>} */
+const devnetChainInfo = {
```

Comment:
TODO: set USDC denom to `ibc/35357FE55D81D88054E135529BB2AEB1BB20D207292775A19BD82D83F27BE9B4`
and use it in `packages/builders/scripts/fast-usdc/start-fast-usdc.build.js`